qa/suites/rbd/basic/cachepool/small.yaml

```diff
@@ -1,5 +1,6 @@
 overrides:
   ceph:
+    create_rbd_pool: true      
```

Comment:
Nit: it's a little odd to see this here. It might be better placed in the `install.yaml`-like tasks (comment applies throughout)

Reply:
Sure will make the changes